example/ios/Podfile.lock

```diff
@@ -381,20 +381,19 @@ DEPENDENCIES:
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
+  trunk:
     - Alamofire
     - boost-for-react-native
+    - CocoaAsyncSocket
     - CocoaLibEvent
+    - Flipper
     - Flipper-DoubleConversion
     - Flipper-Folly
     - Flipper-Glog
     - Flipper-PeerTalk
     - Flipper-RSocket
-    - OpenSSL-Universal
-  trunk:
-    - CocoaAsyncSocket
-    - Flipper
```

Comment:
I'm not sure about these - I had to run `pod install` to get the app up in iOS (@migbot